spec/spec.md

```diff
@@ -1268,6 +1268,54 @@ information to resolve the status of a credential.
 
 </tab-panel>
 
+#### Holder Binding
+Credential often rely on proofs of holder binding for their validity. For
+example, a credential may be bound to a holder through the use of an identifier,
+knowledge of a secret, or via a biometric. A verifier may wish to determine that
+a particular credential, or set of credentials is bound to the credential
+holder. This can help the verifier to determine the legitimacy of the presented
+proofs.
+
+##### Proof of Identifier Control
+A number of credential types include an identifier for the credential subject.
+A verifier may wish to ascertain that the subject identified in the credential
+is the one submitting the proof, or has consented to the proof submission.
+
+One mechanism for providing such proofs is the use of a DID as the identifier
+for the credential subject. DIDs enable an entity to provide a cryptographic
+proof of control of the identifier, usually through a demonstration that the
+holder knows some secret value, such as a private key. The holder can
+demonstrate the same proof of control when presenting the credential. In
+addition to verifying the authenticity and origin of the credential itself, a
+verifier can verify that the presenter of the credential still controls the
+identifier.
+
+##### Link Secrets
+Some credential signatures support the inclusion of holder-provided information
+that becomes incorporated into the signature, but remains hidden from the
+credential issuer. One common use of this capability is known as a link secret.
+A `link secret` is a value known only to the credential holder that is embedded
+in the signature of each issued credential. Upon presentation to a verifier, the
+holder demonstrates knowledge of the random value without revealing it, and
+proves that the same value is contained in every credential included in the
+proof presentation. The verifier can verify that the presenter of the
+credentials knows the link secret, and that the `link secret` is the same in
+each credential.
```

Comment:
wary of putting impl specific ideas into this spec

Reply:
This section is intended to be non-normative and informational. It does touch on some specific aspects of how link secret proofs work, but I tried to make it general enough that any link secret implementation would roughly follow the same steps (much like the proof of control section talks specifically about demonstrating control through proving knowledge of the private key, but is general enough that most schemes would be applicable).
Could you recommend a re-wording that would generalize it enough to be acceptable?

In general, I'm hoping we can settle on a mechanism (ideally one that makes use of submission requirements or input descriptors) that could be used generally by a verifier to indicate its requirements around holder binding. Arguably, we've gotten part of the way there with the `subject_is_holder` flag, but that doesn't cover the cases where 1) a subject gives consent for the presentation (but isn't the holder), or 2) the verifier wants proof that multiple credentials are about the same subject.
I'm also wondering if holder binding is the most accurate term. Or perhaps it's fine, but I'm trying to address more than that here, which would indicate that another term (and another PR) may be required.

Mostly this was my first stab at trying to wrangle a couple of complicated use cases/requirements, so I appreciate the feedback!

Reply:
I have no problem with it, just a call out. I think the approach itself is generic, the addition is great.